lua/orgmode/config/init.lua

```diff
@@ -175,8 +175,28 @@ function Config:setup_mappings(category)
     return
   end
   if not category then
-    utils.keymap('n', self.opts.mappings.global.org_agenda, '<cmd>lua require("orgmode").action("agenda.prompt")<CR>')
-    utils.keymap('n', self.opts.mappings.global.org_capture, '<cmd>lua require("orgmode").action("capture.prompt")<CR>')
+    agenda_key = self.opts.mappings.global.org_agenda
```

Comment:
This can be shortened in both agenda and capture sections a bit like this:
```lua
local agenda_keys = self.opts.mappings.global.org_agenda
if type(agenda_key) == 'string' then
  agenda_keys = { agenda_keys }
end

for _, k in ipairs(agenda_keys) do
  utils.keymap('n', k, '<cmd>lua require("orgmode").action("agenda.prompt")<CR>')
end
```

Reply:
Thank you for your comment! Addressed in 84d1da6